lib/beaker-hostgenerator/data/vmpooler.rb

```diff
@@ -35,6 +35,10 @@ module Vmpooler
           'platform' => 'el-7-x86_64',
           'template' => 'centos-7-x86_64'
         },
+        'cumulus25-64' => {
+          'platform' => 'cumulusvx-25-x86_64',
```

Comment:
@LuvCurves the convention you mention is correct, but Beaker only recognizes specific osnames: https://github.com/puppetlabs/beaker/blob/54b1801d9bb6187796142160d3c0828639c542d5/lib/beaker/platform.rb#L6

So this should probably be `cumulus-25-x86_64`